tests/e2e/010_interfaces_test.go

```diff
@@ -246,39 +240,38 @@ func TestAfPacketInterfaceConn(t *testing.T) {
 	err = req.Delete(
 		afPacket,
 	).Send(context.Background())
-	Expect(err).To(BeNil())
+	Expect(err).ToNot(HaveOccurred())
 
-	Expect(ctx.pingFromVPP(veth2IP)).ToNot(BeNil())
-	Expect(ctx.pingFromMs(msName, afPacketIP)).ToNot(BeNil())
+	Expect(ctx.pingFromVPP(veth2IP)).NotTo(Succeed())
+	Expect(ctx.pingFromMs(msName, afPacketIP)).NotTo(Succeed())
 
 	req = ctx.grpcClient.ChangeRequest()
 	err = req.Update(
 		afPacket,
 	).Send(context.Background())
```

Comment:
Maybe change this to one line too? As you did in the `tests/e2e/020_netalloc_test.go `. Same patterns occurs before and after this line